src/lib/snyk-to-html.ts

```diff
@@ -63,10 +63,9 @@ class SnykToHtml {
                                remediation: boolean,
                                template: string,
                                summary: boolean): Promise<string> {
-    const promisedString = source ? readFile(source, 'utf8') : readInputFromStdin();
-    return promisedString
-      .then(promisedParseJSON)
-      .then(data => processData(data, remediation, template, summary));
+    const jsonData = await (source ? readFile(source, 'utf8') : readInputFromStdin());
```

Comment:
we want to console.log here if !!jsonData

Reply:
the next step does a nice message, do we want another check as well?